examples/caller/configure_vtsi_server.py

```diff
@@ -0,0 +1,71 @@
+from ondewo.vtsi.client import ConfigManager
+
+from ondewo.vtsi.dataclasses.server_configurations_dataclasses import VtsiConfiguration, AudioConfiguration, \
+    AsteriskConfiguration, CaiConfiguration
+
+#     VTSI_SERVER
+VTSI_HOST: str = "0.0.0.0"
+VTSI_PORT: int = 12345
+VTSI_SECURE: bool = False  # if true, VTSI cert is needed
+VTSI_CERT: str = ""
+#     SPEECH2TEXT
+S2T_HOST: str = ""
+S2T_PORT: int = 123
+S2T_TYPE: str = "ONDEWO"
+S2T_LANGUAGE: str = "example_s2t_language"
+#     TEXT2SPEECH
+T2S_HOST: str = ""
+T2S_PORT: int = 123
+T2S_TYPE: str = "ONDEWO"
+T2S_LANGUAGE: str = "example_t2s_language"
+#     ASTERISK
+ASTERISK_HOST: str = "127.0.0.1"
+#     CAI
+CAI_HOST: str = "0.0.0.0"
+CAI_PORT: int = 12345
+CAI_CLIENT_CONFIG: str = "cert/path.json"
+#     PROJECT
+PROJECT_ID: str = "example_project_id"
+
+########################################
+# Full configuration for VTSI, Asterisk, SIP, T2S, S2T modules
+
+# VTSI configuration
+config_voip = VtsiConfiguration(
+    host=VTSI_HOST, port=VTSI_PORT, secure=VTSI_SECURE, cert_path=VTSI_CERT
+)
+
+# S2T and T2S configuration
+config_audio_ondewo = AudioConfiguration(
+    t2s_host=T2S_HOST,
+    t2s_port=T2S_PORT,
+    t2s_type=T2S_TYPE,
+    t2s_language=T2S_LANGUAGE,
+    s2t_host=S2T_HOST,
+    s2t_port=S2T_PORT,
+    s2t_type=S2T_TYPE,
+    s2t_language=S2T_LANGUAGE,
+)
+
+# Asterisk configuration
+config_asterisk = AsteriskConfiguration(
+    host=ASTERISK_HOST,
+)
+
+# CAI/BPI configuration
+# I. Input contexts can be passed here (passed contexts will be used for every calls)
+# or in VtsiClient.start_caller() call
+config_cai = CaiConfiguration(
+    host=CAI_HOST,
+    port=CAI_PORT,
+    cai_project_id=PROJECT_ID,
+    cai_contexts=[],
+)
+
+# ConfigManager to use VTSI client and reach the configs of the different modules
+manager = ConfigManager(
+    config_audio=config_audio_ondewo,
+    config_cai=config_cai,
+    config_voip=config_voip,
+    config_asterisk=config_asterisk,
+)
```

Comment:
i would end all the examples with starting a basic listener

Reply:
so each of the example scripts should work to make a call if you run them. some calls will have different configs, some calls will have contexts, some calls will have multiple targets etc etc